inc/features/blocks/personalization/edit.css

```diff
@@ -0,0 +1,82 @@
+/**
+ * Experience block editor CSS.
+ */
+
+.altis-experience-block-header {
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+	font-size: 16px;
+	background: repeating-linear-gradient(
+		315deg,
+		rgba(0, 0, 0, .05),
+		rgba(0, 0, 0, .05) 10px,
+		rgba(255, 255, 255, .05) 10px,
+		rgba(255, 255, 255, .05) 20px
+	);
+	padding: 14px;
+	margin: 0 -14px -28px;
+	position: relative;
+	top: -14px;
+	display: flex;
+}
+
+.altis-experience-block-header__title {
+	flex: 1;
+	font-weight: bold;
+}
+
+.altis-experience-block-header__toolbar {
+	flex: 0;
+	border: 0;
+	padding: 0;
+	margin: 0;
+	display: flex;
+}
+
+.altis-experience-block-header__toolbar .components-icon-button {
+	padding: 3px 5px;
+	margin-left: 3px;
+}
+
+.altis-variants-toolbar {
+	max-width: 460px;
+	overflow-x: auto;
+}
+
+.altis-variants-toolbar .altis-variant-button {
+	width: auto;
+	margin: 3px 3px 3px 0;
+	height: 30px;
+	line-height: 13px;
+	flex-shrink: 0;
+}
+
+.altis-variants-toolbar .altis-variant-button.is-active,
+.altis-variants-toolbar .altis-variant-button.is-pressed,
+.altis-variants-toolbar .altis-variant-button.is-active:hover,
+.altis-variants-toolbar .altis-variant-button.is-pressed:hover,
+.altis-variants-toolbar .altis-variant-button.is-active:focus,
+.altis-variants-toolbar .altis-variant-button.is-pressed:focus,
+.altis-variants-toolbar .altis-variant-button:not(:disabled):not([aria-disabled="true"]):not(.is-default).is-active,
+.altis-variants-toolbar .altis-variant-button:not(:disabled):not([aria-disabled="true"]):not(.is-default).is-pressed {
```

Comment:
Oof, is there no simpler version of this? Seems like this is mainly just for specificity?

Reply:
Matches the core CSS, and the additional class name is for extra specificity.